server/data.js

```diff
@@ -0,0 +1,7 @@
+const searches = [
+    { id: 1, name: 'Name1', age: 21, email: 'test@gmail.com'},
+    { id: 2, name: 'Name2', age: 43, email: 'test2@gmail.com' },
+    { id: 3, name: 'Name3', age: 84, email: 'test3@gmail.com' },
+];
+
+module.exports = profiles;
```

Comment:
looks like incorrect file has been added, the one cloned has different data.js file and module.exports = searches correctly.